src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerTimerTask.java

```diff
@@ -60,12 +59,7 @@ public void run() {
         if (trigger == null) {
             return;
         }
-        if (StringUtils.isEmpty(trigger.getTriggerConfigURL())) {
-            return;
-        }
-        if (trigger.getJob() != null && !trigger.getJob().isBuildable()) {
-            return;
-        }
```

Comment:
This will risk Jenkins starting to hammer external services more than before, so why is this needed now that there are changes to the latch handling in `Gerrittrigger`?

Reply:
I moved this check into `trigger.updateConfigURL`:1758 since that one will release the workers, otherwise they will be blocked forever. These early returns are the underlying issue of the blocked workers. I also left a comment on the next line. I could improve it if you have any ideas.